packages/create-redwood-app/template/.env.defaults

```diff
@@ -11,3 +11,6 @@ DATABASE_URL=file:./dev.db
 
 # disables Prisma CLI update notifier
 PRISMA_HIDE_UPDATE_MESSAGE=true
+
+# Option to override the current environment's default api-side log level
+# LOG_LEVEL=debug
```

Comment:
Should we document some of the other options, or link to some documentation?
```suggestion
# LOG_LEVEL=debug | trace | off
```

Reply:
Will add options.

The `silent` is important -- it turns logging output off.

 I like the idea of a documentation link and can add that once docs go live. I'll put a placeholder fo when i think the will be now:

> https://redwoodjs.com/docs/logger

And can add a redirect on the website if the url is not correct (and then fix).